packages/website/src/pages/navigation/BreadcrumbsExamples.tsx

```diff
@@ -1,51 +1,78 @@
-                            content: (
-                                <Button>
-                                    <Svg svgName={'add'} className="icon mod-lg mod-align-with-text" />
-                                </Button>
-                            ),
-                        },
-                    ]}
-                />
-            </Section>
-            <Section level={3} title="Breadcrumb header without border bottom">
-                <BreadcrumbHeader
-                    breadcrumb={defaultBreadcrumb}
-                    description="Simple description for the title"
-                    hasBorderBottom={false}
-                />
-            </Section>
-            <Section level={3} title="Breadcrumb header with a section without link">
-                <BreadcrumbHeader
-                    breadcrumb={{...defaultBreadcrumb, links: [link1, {name: 'not a link'}, link2]}}
-                    description="Simple description for the title"
-                    hasBorderBottom={false}
-                >
-                    <Svg svgName="brain" svgClass="icon mod-20 ml1" className="flex" />
-                </BreadcrumbHeader>
-            </Section>
-        </Section>
-    </PlasmaComponent>
+import {ExampleLayout} from '../../building-blocs/ExampleLayout';
+
+const code = `
+    import * as React from "react";
+    import {BreadcrumbHeader, IBreadcrumbProps} from '@coveord/plasma-react';
+
+    const defaultBreadcrumb: IBreadcrumbProps = {
+        title: {
+            text: 'Charmeleon',
+        },
+        links: [{
+          name: 'Pikachu',
+          link: 'https://www.google.ca/?q=pikachu',
+        }],
+    };
+
+
+    export default () => 
+      <BreadcrumbHeader
+          breadcrumb={defaultBreadcrumb}
+          description="Simple description for the title"
+          hasBorderBottom={false}
+      />;
+`;
+
+const complex = `
+    import * as React from "react";
+    import {BreadcrumbHeader, IBreadcrumbProps} from '@coveord/plasma-react';
+
+    const defaultBreadcrumb: IBreadcrumbProps = {
+        title: {
+          text: 'Charmeleon',
+          documentationLink: {
+          target: '_blank',
+          svg: {
+            svgName: 'help',
+            svgClass: 'documentation-link icon mod-20',
+            className: 'flex',
+          },
+          tooltip: {
+            title: "I'm a tooltip!",
+            placement: 'bottom',
+            container: 'body',
+          },
+        },
+        },
+        links: [{
+          name: 'Pikachu',
+          link: 'https://www.google.ca/?q=pikachu',
+        }],
+    };
+
+    export default () => (
+      <BreadcrumbHeader
+        breadcrumb={defaultBreadcrumb}
+        description="Simple description for the title"
+        tabs={[
+            {groupId: 'example2', id: 'tab1', title: 'Digimon'},
+            {groupId: 'example2', id: 'tab2', title: 'Beyblade'},
+            {groupId: 'example2', id: 'tab3', title: 'Pokemon'},

```

Comment:
Should have `DragonBall` but whatever

Reply:
It should but Pokemon was already there :grimacing: